src/storage/exec/IndexScanNode.h

```diff
@@ -153,6 +154,10 @@ class IndexScanNode : public IndexNode {
    */
   std::unique_ptr<kvstore::KVIterator> iter_;
   nebula::kvstore::KVStore* kvstore_;
+  /**
+   * @brief if index contain nullable field or not
```

Comment:
Why change this line's order, keep align with new added initilize?

Reply:
Yeah, just want to keep align the parameter order in constructor and member initialize list.